src/components/Card/README.md

```diff
@@ -12,6 +12,7 @@ By default the Card will pull the theme from the site (site colors),
 | MainMediaColor  | string   | color-5 20% opacity | The color of the image container instead of the default color |
 | MainBorderColor  | string   | color-5 | The color of the border instead of the default color |
 | MainBorderWidth  | string   | 1px | The width of the border instead of the default width |
+| MainInfoSidePadding  | string   | 32px | The width of Stack layout Side padding |
```

Comment:
Maybe we should call it `StackedInfoSidePadding`?
Because it might confuse users that it isn't affecting horizontal cards.
Or maybe this should apply on both types?

Reply:
@jonathanadler renamed